package.json

```diff
@@ -55,6 +55,7 @@
     "dotenv-flow": "^3.2.0",
     "eslint": "^8.25.0",
     "eslint-config-next": "^13.4.13",
+    "eslint-plugin-prettier": "^5.0.0",
```

Comment:
We shouldn't need to have that as a dependency here as it is already a dependency of `@inrupt/eslint-config-base`

Reply:
That's what I thought too, but ESLint complains about this missing if not installed, and the same setup is present in `solid-client`. Let me see if that could be related to your next comment.

Reply:
Well removing both dependencies doesn't cause the issue to come back. I also opened https://github.com/inrupt/solid-client-js/pull/2149 to do the same in `solid-client`

Reply:
See https://github.com/inrupt/solid-client-authn-js/pull/3100/commits/cdf7b4252cd52a74d5bcb3f2948f6939f931f119